workers.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_autoscaling_group" "workers" {
     "-",
     compact(
       [
-        aws_eks_cluster.this[0].name,
+        coalesce(lookup(var.worker_groups[count.index], "asg_name_prefix", local.workers_group_defaults["asg_name_prefix"]), aws_eks_cluster.this[0].name),
```

Comment:
Can you also make this change in `workers_launch_template.tf`?

Reply:
@max-rocket-internet done.

Reply:
And use something like in worker.tf and workers_launch_template.tf
```hcl
lookup(var.worker_groups[count.index], "asg_name_prefix", local.workers_group_defaults["asg_name_prefix"]),
```